### PR TITLE
Pop-up and Options page

### PR DESCRIPTION
**Pull request by YuvrajSharma07**
- **What does this PR do?**
The pop-up window has been improved with a little bit of responsive behaviour, along with font-family changes all over the doc. The options section has also been modified.
- **Any background context you want to provide?**
Addresses issue #1  and #9 .
- **Screenshots and/or Live Demo**
demo

### CODE REVIEW

**Comment by prateek3255** [comment]
This seems nice, but the options page needs to be a separate page because it can also be accessed from the extension menu. 
image

But I really liked your idea, so I would suggest moving the options code to popup. And rename it something like `Recently Saved Scrrolls`.

Also just use something like [this](https://stackoverflow.com/a/36961138) to reverse the object, so that we have the recently saved URLs first. And slice it to show the top 5 URLs, since we are going to show all of them in the options page anyway.

**Comment by YuvrajSharma07** [comment]
sure thing! I had something similar in mind, just needed some validation.

**Comment by prateek3255** [comment]
@YuvrajSharma07 I have updated the master branch so please update your branch accordingly. Also, let me know if you're stuck anywhere.

**Comment by YuvrajSharma07** [comment]
Yeah! I have even styled the options page, added some bar indicating the percentage as well. Will do the popup object reverse thing and commit soon. Here is a snippet
Screenshot 2019-10-08 at 3 17 35 PM

File: options.js [hunk 1]
```diff
@@ -1,4 +1,18 @@
 {
+<<<<<<< HEAD
```
Comment (prateek3255) [inline options.js]: Please remove these merge conflict indicators

File: options.js [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+    chrome.storage.local.get("scroll-mark", function(result) {
+        urls = result["scroll-mark"];
+        for (let url in urls) {
+            i++;
+            let div = document.createElement("div");
+            div.innerHTML = "<a href=" + url + " class='links' title="+url+">" + i + ") " + url + "..." + "</a>";
+            list.appendChild(div);
+        }
+    });
+=======
```
Comment (prateek3255) [inline options.js]: ☝️

File: options.js [hunk 3]
```diff
@@ -14,4 +28,20 @@
       list.appendChild(div);
     }
   });
+>>>>>>> devfolioco/master
```
Comment (prateek3255) [inline options.js]: ☝️

File: save.js [hunk 4]
```diff
@@ -1,4 +1,17 @@
 {
+<<<<<<< HEAD
```
Comment (prateek3255) [inline save.js]: ☝️

File: save.js [hunk 5]
```diff
@@ -1,4 +1,17 @@
...
+        const scrollMarkData = data["scroll-mark"];
+        const offset = window.pageYOffset;
+        const newData = scrollMarkData
+        ? { ...scrollMarkData, [url]: offset }
+        : { [url]: offset };
+        console.log(offset, scrollMarkData[url], newData[url]);
+        chrome.storage.local.set({ "scroll-mark": newData }, () => {
+            chrome.runtime.sendMessage("setActive");
+        });
+=======
```
Comment (prateek3255) [inline save.js]: ☝️

File: save.js [hunk 6]
```diff
@@ -12,6 +25,6 @@
     console.log(offset, scrollMarkData[url], newData[url]);
     chrome.storage.local.set({ "scroll-mark": newData }, () => {
       chrome.runtime.sendMessage("setActive");
+>>>>>>> devfolioco/master
```
Comment (prateek3255) [inline save.js]: ☝️

**Comment by prateek3255** [comment]
Also, I am planning to go forward with #15 for the options page

**Comment by YuvrajSharma07** [comment]
Yeah, completely forgot about that file.

Modified the options page to maintain design consistency with the popup, otherwise it's your call :)

If you don't mind, I will finish the code for popup by Saturday, been a little hooked up with other work stuff..

**Comment by prateek3255** [comment]
You did not removed the merge conflict indicators again.

File: options.css [hunk 7]
```diff
@@ -1,23 +1,106 @@
...
+    overflow-x: hidden;
+}
+a, a:visited {
+    text-decoration: none;
+    color: #006699
+}
+=======
 
 
+>>>>>>> devfolioco/master
```
Comment (prateek3255) [inline options.css]: ☝️

**Comment by YuvrajSharma07** [comment]
demo 1
A demo of the few changes i made in the popup, the reverse object isnt too prominent, because of the issue #42, but it is functional, you may modify the options page as you may feel like :)

I took the liberty of using a different font to make it all look better, do remark on that too

**Comment by YuvrajSharma07** [comment]
@prateek3255

**Comment by prateek3255** [comment]
Moving in a different direction, so won't be able to merge it for now. Thanks for experimenting with this.

**Pull request closed without merging**